app/policies/wiki_policy.rb

```diff
@@ -0,0 +1,17 @@
+class WikiPolicy < ApplicationPolicy
+  attr_reader :user, :wiki
+
+  def initialize(user, wiki)
+    @user = user
+    @wiki = wiki
+  end
+
+  def new
```

Comment:
you're trying to overwrite the pundit policy for the new action, but the syntax that pundit uses is the name of the action followed by a question mark (e.g. `new?`).  so you're not able to overwrite because you haven't named the policy correctly.  once you do, it should work.

the reason they use new? is because it's a question asking if this user should be allowed to the new action for this resource (i.e. "should user be authorized for new?") and the returned value will be true or false.  it doesn't have to be a question mark, but the developers of the Pundit gem felt it makes sense when you think about it this way.  this is a common pattern - if we're basically asking whether something is true or false, we often name the method with a question mark to indicate that.